### PR TITLE
ddl: Fix the bug that showing status row count is 0

### PR DESCRIPTION
**Pull request by zimulala**
fix issue #2691

### CODE REVIEW

File: ddl/index_change_test.go [hunk 1]
```diff
@@ -103,6 +103,9 @@ func (s *testIndexChangeSuite) TestIndexChange(c *C) {
 				checkErr = errors.Trace(err)
 			}
 		case model.StatePublic:
+			if job.RowCount != 3 {
+				checkErr = errors.Errorf("job's row count %d != 3", job.RowCount)
```
Comment (hanfei1991) [inline ddl/index_change_test.go]: Why not use `c.Check` ?

File: ddl/reorg.go [hunk 2]
```diff
@@ -61,16 +70,22 @@ func (d *ddl) runReorgJob(f func() error) error {
...
 	case err := <-d.reorgDoneCh:
-		log.Info("[ddl] run reorg job done")
 		d.reorgDoneCh = nil
+		// Update a job's RowCount.
+		job.RowCount = d.getReorgRowCount()
+		log.Infof("[ddl] run reorg job done, handled row count %v", job.RowCount)
+		d.setReorgRowCount(0)
 		return errors.Trace(err)
 	case <-d.quitCh:
 		log.Info("[ddl] run reorg job ddl quit")
```
Comment (hanfei1991) [inline ddl/reorg.go]: Why not log row count ?

File: ddl/reorg.go [hunk 3]
```diff
@@ -61,16 +70,22 @@ func (d *ddl) runReorgJob(f func() error) error {
...
+		d.setReorgRowCount(0)
+		// We return errWaitReorgTimeout here too, so that outer loop will break.
 		return errWaitReorgTimeout
 	case <-time.After(waitTimeout):
-		log.Infof("[ddl] run reorg job wait timeout :%v", waitTimeout)
-		// if timeout, we will return, check the owner and retry to wait job done again.
+		log.Infof("[ddl] run reorg job wait timeout %v, handled row count %v", waitTimeout, job.RowCount)
+		// Update a job's RowCount.
+		job.RowCount = d.getReorgRowCount()
+		// If timeout, we will return, check the owner and retry to wait job done again.
```
Comment (hanfei1991) [inline ddl/reorg.go]: Why not set 0 ?

**Comment by coocood** [comment]
LGTM

**Pull request merged**